client/my-sites/checkout/composite-checkout/components/prepurchase-notices/index.jsx

```diff
@@ -69,6 +76,31 @@ const PrePurchaseNotices = () => {
 		return products.filter( ( p ) => ! p.expired );
 	} );
 
+	const akismetProductSlugThatOverlapsCartProduct = useMemo( () => {
+		const productSlugInCart = cartItemSlugs.find( ( productSlug ) =>
+			isAkismetProduct( { productSlug } )
+		);
+		const akismetPurchases = userActivePurchases.filter( isAkismetProduct );
```

Comment:
Wondering if we have a way to determine if the purchase is a "site-less" purchase when filtering. If it's possible to get only "site-less" Akismet purchases, that would be preferable, but not a blocker if not - this would be some edge case handling.

Reply:
Yes, we can use `domain` property from Purchase object